patches/chrome-browser-ui-views-sharing_hub-sharing_hub_bubble_view_impl.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/sharing_hub/sharing_hub_bubble_view_impl.cc b/chrome/browser/ui/views/sharing_hub/sharing_hub_bubble_view_impl.cc
+index 515fb2ce8d47f75a09283438198691fdf78328e2..3214caa20c0b6b6833c2ed9ed63a0aef90708682 100644
+--- a/chrome/browser/ui/views/sharing_hub/sharing_hub_bubble_view_impl.cc
++++ b/chrome/browser/ui/views/sharing_hub/sharing_hub_bubble_view_impl.cc
+@@ -163,6 +163,7 @@ void SharingHubBubbleViewImpl::PopulateScrollView(
+     view->SetGroup(kActionButtonGroup);
+   }
+ 
++  BRAVE_SHARING_HUB_BUBBLE_VIEW_IMPL_POPULATE_SCROLL_VIEW
```

Comment:
if possible, maybe we can use this?
```
BraveSharingHubBubbleViewImpl::PopulateScrollView() {
  SharingHubBubbleViewImpl::PopulateScrollView();
  if (share_link_label_) {
    action_list_view->RemoveChildView(share_link_label_);
    MaybeSizeToContents();
    Layout();
  }
}
```

Reply:
Hmm, I think it's difficult because we should also remove below separator view.
It seems difficult from subclass.